packages/components/src/docs/Migration/ui-image.mdx

```diff
@@ -0,0 +1,67 @@
+# Migration Guide: `ui-image`
+
+The new `sd-media` and `sd-copyright` are designed to replace the `ui-image`. Rather than offering a complex component structure, they primarily focus on providing CSS styles.
```

Comment:
Please make more clear, what the two components do and hwo they should/could be used. See e. g. https://solid-design-system.fe.union-investment.de/x.x.x/storybook/?path=/docs/styles-sd-media--docs

It could be even worth to show some example code here. But do not feel forced to do so.

But definitely describe better, what each CSS does.